<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmbedderOptions</class>
 <widget class="QWidget" name="EmbedderOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>373</width>
    <height>140</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Embedder options</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidgetEmbedderOptions">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabSnapshots">
      <attribute name="icon">
       <iconset resource="qmc2.qrc">
        <normaloff>:/data/img/camera.png</normaloff>:/data/img/camera.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Snapshots</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <widget class="QListWidget" name="listWidgetSnapshots">
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="movement">
          <enum>QListView::Static</enum>
         </property>
         <property name="flow">
          <enum>QListView::LeftToRight</enum>
         </property>
         <property name="isWrapping" stdset="0">
          <bool>true</bool>
         </property>
         <property name="resizeMode">
          <enum>QListView::Adjust</enum>
         </property>
         <property name="viewMode">
          <enum>QListView::IconMode</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QToolButton" name="toolButtonTakeSnapshot">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Take a snapshot of the current window content -- hold to take snapshots repeatedly (every 100ms)</string>
         </property>
         <property name="icon">
          <iconset resource="qmc2.qrc">
           <normaloff>:/data/img/camera.png</normaloff>:/data/img/camera.png</iconset>
         </property>
         <property name="autoRepeat">
          <bool>true</bool>
         </property>
         <property name="autoRepeatDelay">
          <number>200</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabMovies">
      <attribute name="icon">
       <iconset resource="qmc2.qrc">
        <normaloff>:/data/img/movie.png</normaloff>:/data/img/movie.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Movies</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tabNetplay">
      <attribute name="icon">
       <iconset resource="qmc2.qrc">
        <normaloff>:/data/img/world.png</normaloff>:/data/img/world.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Netplay</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidgetEmbedderOptions</tabstop>
  <tabstop>toolButtonTakeSnapshot</tabstop>
  <tabstop>listWidgetSnapshots</tabstop>
 </tabstops>
 <resources>
  <include location="qmc2.qrc"/>
 </resources>
 <connections/>
</ui>
